libraries/botbuilder-dialogs/package.json

```diff
@@ -47,7 +48,7 @@
     "build-docs": "typedoc --theme markdown --entryPoint botbuilder-dialogs --excludePrivate --includeDeclarations --ignoreCompilerErrors --module amd --out ..\\..\\doc\\botbuilder-dialogs .\\lib\\index.d.ts --hideGenerator --name \"Bot Builder SDK - Dialogs\" --readme none",
     "build:rollup": "yarn clean && yarn build && api-extractor run --verbose --local",
     "clean": "rimraf _ts3.4 lib tsconfig.tsbuildinfo",
-    "depcheck": "depcheck --config ../../.depcheckrc",
+    "depcheck": "depcheck --config ../../.depcheckrc --ignores uuid,sinon",
```

Comment:
Do we need to add sinon?

Reply:
Yes, after adding uuid, sinon started to show up too, we took the ignore line for another project with the same issue so we have consistency